packages/synapse-react-client/src/utils/SynapseError.ts

```diff
@@ -1,6 +0,0 @@
-/**
- * Error message returned by the Synapse Backend
- */
-export type SynapseError = {
```

Comment:
This was just used in a few places in test code. Removed in favor of using the `ErrorResponse` generated using the OpenAPI schema